http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/ReactiveRestClientIT.java

```diff
@@ -202,22 +202,24 @@ public void checkSuffixPriority() {
     @Test
     public void checkProcessPathBeforeSubResources() {
         // should result in sending GET root/method/sub/sub/subsub
-        String result = app.given().get("/plain-root/root/method/sub/subsub").then()
+        String id = "3479dd56-02e9-4222-94fe-6a13cd065195";
+        String result = app.given().delete("clients/myRealm/clientResource/" + id).then()
                 .statusCode(HttpStatus.SC_OK)
                 .extract().asString();
 
-        assertEquals("root/method/sub/sub/subsub", result);
+        assertEquals("/clients/myRealm/authz/resource-server/permission/resource/" + id, result);
     }
 
     @Tag("QUARKUS-2741")
     @Test
     public void checkProcessPathBeforeSubResourcesManualRestClientBuild() {
+        String id = "3479dd56-02e9-4222-94fe-6a13cd065195";
```

Comment:
What is the significance of this line? What will happen if we replace it with "d34d-b34at"? Or with "1"? Or with "some-id"?

Reply:
No, could be any ID, I have changed the name of the variable in order to make it clear